tasks/src/main/scala/dagr/tasks/picard/DownsampleSam.scala

```diff
@@ -34,7 +34,7 @@ import scala.collection.mutable.ListBuffer
 
 object DownsamplingStrategy extends Enumeration {
   type DownsamplingStrategy = Value
-  val HighAccuracy, ConstantMemory, Chained = Value
+  val ConstantMemory, Chained, HighAccuracy = Value
```

Comment:
It really bothered me that the match was a different order than in the enumeration. I agree generally it's not a good idea.